src/Message.ts

```diff
@@ -420,3 +396,27 @@ export async function decodeContent(contentBytes: Uint8Array, client: Client) {
 
   return { content, contentType, error }
 }
+
+function conversationReferenceToConversation(
```

Comment:
Moved this out of the class so that it doesn't show in the typedocs. Feels internal and not useful to the public.